test/test_module.yaml

```diff
@@ -10,8 +10,8 @@ paths:
             title: Wikimedia testing APIs
             x-is-api-root: true
           paths:
-            /service/test/{title}{/revision}:
-              get:
+            /service/test/{title}:
+              get: &service_test_title
```

Comment:
Better to be consistent and have the ref be for `/service/test/{title}` instead of `get`